content/docs/introduction/glossary.md

```diff
@@ -108,11 +108,17 @@ write into a format the other system can understand.
 
 A remote write endpoint is what Prometheus talks to when doing a remote write.
 
+### Sample
+
+A sample is a single value at a point in time in a time series.
+
+In Prometheus, each sample consists of a float64 representing some metric and a millisecond-precision timestamp.
+
 ### Silence
 
 A silence in the Alertmanager prevents alerts, with labels matching the silence, from
 being included in notifications.
 
 ### Target
 
-A target is the definition of an object to scrape. For example, what labels to apply, any authentication required to connect, or other information that defines how the scrape will occur.
+A target is the definition of an object to scrape. For example, what labels to apply, any authentication required to connect, or other information that defines how the scrape will occur.
```

Comment:
Could you configure your editor to always end files with a newline character (standard on Unix and otherwise causes unnecessary diffs like this one)?